packages/odyssey-react/src/components/VisuallyHidden/VisuallyHidden.test.tsx

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { render } from "@testing-library/react";
+import VisuallyHidden from ".";
+
+const content = "This string is visually hidden, but screen-reader accessible.";
+
+describe("VisuallyHidden", () => {
+  it("renders invisibly into the document", () => {
+    const { getByText } = render(
+      <VisuallyHidden>{content}</VisuallyHidden>
+    );
+
+    expect(getByText(content)).toBeVisible();
```

Comment:
`.toBeVisible` ensures that this isn't hidden from the DOM (e.g. `hidden` or `display: none;`).

`.toHaveStyle` is something I would normally avoid, but it feels like business logic in the case of VisuallyHidden. Rather than duplicate the entire style block here, I'm pointing to the most esoteric property declaration.

@jeffbelser-okta @andrewberg-okta @arnoldsandoval-okta please lmk if there's a better approach here.

Reply:
Yes there is some nuance to the `toBeVisible` assertion but I think the tests as yo have them here are OK. In a perfect world we'd used VRT or an integration test to assert on the content being screen reader only rather than having to assert against a single style here.